src/bin/lib/libs/ini.h

```diff
@@ -486,7 +486,6 @@ ini_t* ini_create( void* memctx )
     ini->sections[ 0 ].name[ 0 ] = '\0'; 
     ini->sections[ 0 ].name_large = 0;
     ini->properties = (struct ini_internal_property_t*) INI_MALLOC( ini->memctx, INITIAL_CAPACITY * sizeof( ini->properties[ 0 ] ) );
-	memset(ini->properties, 0, INITIAL_CAPACITY * sizeof(ini->properties[0]));
```

Comment:
This seems like a change that we introduced. I guess with a reason.

Reply:
isn't that https://github.com/mattiasgustavsson/libs/commit/5d00587c897f7992c8bd948ca1f44938325bade4#diff-3d315c8a064a9c199059040b5edc4dc7 ?

Reply:
mmm that said I'm not sure after all, did our change come from the security work stream and review?

Reply:
thanks for the link to https://github.com/citusdata/citus-ha/commit/d3ccd755dc4c534e7356f2508f42a1597e7be934 ; cancelled my change.